.github/workflows/README.md

```diff
@@ -1,12 +1,27 @@
 Actions:
 
+You have an automerge action which has not been explained. Does this allow you to merge to automatically when the ci completes successfully? What is the purpose of this action?
+Also what is this line doing ` uses: "pascalgn/automerge-action@v0.9.0"`? Probably want to include a link https://github.com/pascalgn/automerge-action
+
 -    `continuous-integration-workflow.yml`: runs `cmake`, `make`, `ctest` and `make install` etc. for many different compiler,
      distro / gromacs combinations and special builds for minimal, internal gromacs and coverage
      -    common setup parts done by votca/actions/setup action that can be found here: https://github.com/votca/votca/actions 
           it mainly generates the right cmake arguments (`cmake_args`) for the combination of build parameters
      -    deploy of website and doxygen is happening in `deploy.sh` (push happens for master only)
+
+For the continuous workflow action is this scheduled to run every friday `  - cron:  '0 5 * * FRI'` if so it would probably be a good idea to have a badge displaying whether it is passing or not. 
+
+Can you explain what the id setup is being used for, is this just a unique identifier so you can store ccache in seperate folders to have a fast build? 
```

Comment:
Correct, see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs for docs.

Reply:
Added comment in workflow.